src/commands/Games/hungergames.ts

```diff
@@ -179,7 +178,7 @@ export default class extends SkyraCommand {
 		const proceed = t(LanguageKeys.Commands.Games.HungerGamesResultProceed);
 		const panels = chunk(results, 5);
 
-		const texts = panels.map((panel) => `__**${header}:**__\n\n${panel.map((text) => `- ${text}`).join('\n')}\n\n_${proceed}_`);
+		const texts = panels.map((panel) => `__**${header}:**__\n\n${panel.map((text) => `- ${text}`).join('\n')}\n\n_${proceed}_`) as string[];
```

Comment:
Is this cast needed?

Reply:
Yes, because TypeScript 4.2.0 considers `texts` an array of templated types, on similar code:

![image](https://user-images.githubusercontent.com/24852502/105019510-7447b880-5a46-11eb-9a59-d89bc6703037.png)

Reply:
Ohhhh, I see